yarn-project/end-to-end/src/e2e_l1_publisher/e2e_l1_publisher.test.ts

```diff
@@ -749,7 +749,7 @@ describe('L1Publisher integration', () => {
 
       await publisher.enqueueProposeL2Block(block, CommitteeAttestationsAndSigners.empty(), Signature.empty());
       await publisher.enqueueGovernanceCastSignal(
-        EthAddress.random(),
+        l1ContractAddresses.rollupAddress,
```

Comment:
Hmm I'm not sure this is correct. The first parameter is to `enqueueGovernanceCastSignal` is the governance payload, not the rollup. If we need a stable address in order to vote for it maybe we can generate one via `EthAddress.random()` and store it in a variable?

Reply:
In this case we just need any contract with code. `enqueueGovernanceCastSignal` should return true and then actual send should fail